doc/projects_list.rst

```diff
@@ -4,7 +4,7 @@ Who uses Sphinx-Gallery
 
 Here is a list of projects using `sphinx-gallery`.
 
-* `Sphinx-Gallery <https://sphinx-gallery.github.io/auto_examples/index.html>`_
+* :ref:`Sphinx-Gallery <sphx_glr_auto_examples>`
 * `Scikit-learn <http://scikit-learn.org/dev/auto_examples/index.html>`_
```

Comment:
At some point we should update the others we also have intersphinx references for ([sklearn, matplotlib](https://github.com/sphinx-gallery/sphinx-gallery/blob/master/doc/conf.py#L305-L309))